specification.md

```diff
@@ -688,8 +688,30 @@ JobSpec()
 
 Constructs an empty `JobSpec`.
 
-#### Methods
+<div class="lang-bindings">
+
+__Python__:
+
+In Python, the following additional constructor is defined:
```

Comment:
The argument-less constructor and this constructor are mutually exclusive in Python, right?  Do we want to also put the empty JobSpec constructor in this lang-bindings class?  Or maybe tweak the wording:

```suggestion
In Python, the following constructor is defined in lieu of the argument-less constructor:
```

Reply:
Thanks for catching this. Yes, I think the wording should be updated. The all-keyword-arg constructor can be called as nullary, but the text should make it clear that that's not an "additional constructor".